### PR TITLE
added emissions and emission factors for SSP3

### PR DESCRIPTION
**Pull request by nicobauer**
I added the implementation of the SSP3 emissions and emission factors (copy-paste of SSP2) and set everything to CLE (Current Legislation) in all regions and all times.

### CODE REVIEW

**Review by LaviniaBaumstark — APPROVED** [review]

File: R/calcEmissionFactors.R [hunk 1]
```diff
@@ -305,7 +305,45 @@ calcEmissionFactors <- function(subtype = "emission_factors", sectoral_resolutio
...
+  ef[r_L, 2100, "SSP3"]       <- pmin(setYears(ef[r_L, 2050, "SSP3"]),
+                                      setYears(allocate_c2r_ef(ef_eclipse, r_L, select_weu, 2030, "CLE")))         # 2100: CLE30 WEU
+
+  # Emissions
+  # High-Medium income countries with strong pollution policies in place
+  emi[r_HMStrong, 2030, "SSP3"] <- emissions_exogenous[r_HMStrong, 2030, "CLE"]                                               # 2030: CLE30
+  emi[r_HMStrong, 2050, "SSP3"] <- pmin(setYears(emi[r_HMStrong,        2030, "SSP3"]),
+                                        setYears(emissions_exogenous[r_HMStrong, 2030, "CLE"]))                                # 2050: CLE30
+  emi[r_HMStrong, 2100, "SSP3"] <- pmin(setYears(emi[r_HMStrong,        2050, "SSP3"]),
+                                        setYears(emissions_exogenous[r_HMStrong, 2030, "CLE"] ))                           # 2100: Lowest CLE30 or lower -> 0.8*CLE30
```
[the remark this answers is not in the document]
Comment (merfort) [reply R/calcEmissionFactors.R:381]: (so it should be removed from the comment as well)

File: R/calcEmissionFactors.R [hunk 2]
```diff
@@ -305,7 +305,45 @@ calcEmissionFactors <- function(subtype = "emission_factors", sectoral_resolutio
...
+  # High-Medium income countries with lower emissions goals
+  emi[r_HMRest, 2030, "SSP3"]  <- emissions_exogenous[r_HMRest, 2030, "CLE"]                                                  # 2030: CLE30
+  emi[r_HMRest, 2050, "SSP3"]  <- pmin(setYears(emi[r_HMRest,       2030, "SSP3"]),
+                                       setYears(emissions_exogenous[r_HMRest, 2030, "CLE"]))                                  # 2050: Min CLE30 -> CLE30
+  emi[r_HMRest, 2100, "SSP3"]  <- pmin(setYears(emi[r_HMRest, 2050, "SSP3"]),
+                                       setYears(emissions_exogenous[r_HMRest, 2030, "CLE"] ))                              # 2100: CLE30 WEU -> 0.8*CLE30
```
Comment (merfort) [inline R/calcEmissionFactors.R:387]:
```suggestion
                                       setYears(emissions_exogenous[r_HMRest, 2030, "CLE"] ))                              # 2100: CLE30 WEU
```

File: R/calcEmissionFactors.R [hunk 3]
```diff
@@ -305,7 +305,45 @@ calcEmissionFactors <- function(subtype = "emission_factors", sectoral_resolutio
...
+  # low income countries
+  emi[r_L, 2030, "SSP3"]       <- setYears(emissions_exogenous[r_L, 2020, "CLE"])                                           # 2030: CLE20
+  emi[r_L, 2050, "SSP3"]       <- pmin(setYears(emi[r_L, 2030, "SSP3"]),
+                                       setYears(emissions_exogenous[r_L, 2030, "CLE"]))                                     # 2050: Min CLE30 -> CLE30
+  emi[r_L, 2100, "SSP3"]       <- pmin(setYears(emi[r_L, 2050, "SSP3"]),
+                                       setYears(emissions_exogenous[r_L, 2030, "CLE"] ))                                # 2100: CLE30 WEU -> 0.95*CLE30
```
Comment (merfort) [inline R/calcEmissionFactors.R:393]:
```suggestion
                                       setYears(emissions_exogenous[r_L, 2030, "CLE"] ))                                # 2100: CLE30 WEU
```

**Comment by nicobauer** [comment]
I also do not know why that change was introduced and hardcoded. I guess this was Sebastian. I did not follow the log backwards.

**Pull request merged**